### PR TITLE
Returning Evaluation results

### PR DESCRIPTION
**Pull request by AmrMKayid**
Return the evaluation results dictionary if `return_resutls` is set to `True`

### CODE REVIEW

File: mteb/evaluation/MTEB.py [hunk 1]
```diff
@@ -168,7 +168,7 @@ def load_tasks_data(self):
             logger.info(f"\n# Loading dataset for {task.description['name']}")
             task.load_data()
 
-    def run(self, model, verbosity=1, output_folder="results/result", eval_splits=None, **kwargs):
+    def run(self, model, verbosity=1, output_folder="results/result", eval_splits=None, return_resutls=False, **kwargs):
```
Comment (amorisot) [inline mteb/evaluation/MTEB.py:171]:
```suggestion
    def run(self, model, verbosity=1, output_folder="results/result", eval_splits=None, return_results=False, **kwargs):
```

**Comment by NouamaneTazi** [comment]
Apologies for the delay. Thank you for the contribution! I think many users would find this handy, would you like to add a small example of how to use the flag in the README?

**Comment by Muennighoff** [comment]
t doesn't hurt always returning them, so I would remove the kwarg & always return them; What do you think @NouamaneTazi ?

**Comment by NouamaneTazi** [comment]
True, I don't see a problem with that neither. As long as we update the related docs.

**Comment by Muennighoff** [comment]
@AmrMKayid If you could make it the default & update the docstring of `run` that would be amazing!

**Comment by AmrMKayid** [comment]
@NouamaneTazi @Muennighoff Thank you very much for the feedback! I have made it the default and updated the docs :))

**Review by NouamaneTazi — APPROVED** [review]

**Comment by NouamaneTazi** [comment]
Amazing! Thanks for the clean PR :)

**Pull request merged**